requests_kerberos/exceptions.py

```diff
@@ -10,3 +10,8 @@
 
 class MutualAuthenticationError(RequestException):
     """Mutual Authentication Error"""
+
+class KerberosExchangeError(RequestException):
+    """Kerberos Exchange Failed Error"""
+    def __init__(self, *args, **kwargs):
+        super(KerberosExchangeError, self).__init__(*args, **kwargs)
```

Comment:
Urg, gotta stop doing this past 1am- more vestigial crap from a different version of that error I missed removing.